src/db/oracledb/jobs-dao.js

```diff
@@ -56,4 +120,269 @@ const getJobByJobId = async (internalId, jobId) => {
+      && flattenedBody.salaryInformationReleaseInd !== undefined) {
+    flattenedBody.salaryInformationReleaseInd = flattenedBody
+      .salaryInformationReleaseInd
+      .toString();
+  }
+
+  _.merge(binds, _.pick(flattenedBody, [
+    'positionNumber',
+    'suffix',
+    'effectiveDate',
+    ...additionalFields || [],
+  ]));
+
+  return binds;
+};
+
+/**
+ * Updates job record
+ *
+ * @param {object} connection oracledb connection
+ * @param {string} osuId OSU ID of a person
+ * @param {object} body Request body
+ * @returns {string} Query result, null if success
+ */
+const updateJob = async (connection, osuId, body) => {
+  const binds = standardBinds(osuId, body, [
+    'hourlyRate',
+    'appointmentPercent',
+    'personnelChangeDate',
+    'hoursPerPay',
+    'annualSalary',
+    'fullTimeEquivalency',
+    'changeReason_code',
+    'salary_annual',
+  ]);
+
+  const { outBinds: { result } } = await connection.execute(contrib.updateJob(binds), binds);
+  return result;
+};
+
+/**
+ * Terminates a job record
+ *
+ * @param {object} connection oracledb connection
+ * @param {string} osuId OSU ID of a person
+ * @param {object} body Request body
+ * @returns {string} Query result, null if success
+ */
+const terminateJob = async (connection, osuId, body) => {
+  const binds = standardBinds(osuId, body, ['changeReason_code']);
+
+  const { outBinds: { result } } = await connection.execute(contrib.terminateJob(), binds);
+  return result;
+};
+
+/**
+ * Updates labor distribution for a job
+ *
+ * @param {object} connection oracledb connection
+ * @param {string} osuId OSU ID of a person
+ * @param {object} body Request body
+ * @returns {string} Query result, null if success
+ */
+const updateLaborChangeJob = async (connection, osuId, body) => {
+  const binds = standardBinds(osuId, body, ['laborDistribution']);
+
+  const { outBinds: { result } } = await connection.execute(contrib.updateLaborChangeJob(), binds);
+  return result;
+};
+
+/**
+ * Updates student job records
+ *
+ * @param {object} connection oracledb connection
+ * @param {string} osuId OSU ID of a person
+ * @param {object} body Request body
+ * @returns {string} Query result, null if success
+ */
+const studentUpdateJob = async (connection, osuId, body) => {
+  const binds = standardBinds(osuId, body, studentBinds);
+
+  const result = await connection.execute(contrib.studentUpdateJob(binds), binds);
+  return result.outBinds.result;
+};
+
+/**
+ * Creates student job records
+ *
+ * @param {object} connection oracledb connection
+ * @param {string} osuId OSU ID of a person
+ * @param {object} body Request body
+ * @returns {string} Query result, null if success
+ */
+const studentCreateJob = async (connection, osuId, body) => {
+  const binds = standardBinds(osuId, body, studentBinds);
+
+  const { outBinds: { result } } = await connection.execute(contrib.studentCreateJob(binds), binds);
+  return result;
+};
```

Comment:
`studentUpdateJob` and `studentCreateJob` can be combined as you did for `graduateJob`.